tensorflow-base-cpu-gpu-linwin/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2.9.1" %}
+{% set version = "2.10.0" %}
 {% set major_minor = version.rsplit('.', 1)[0] %}
 
 # This is the recipe for the "mkl", "eigen" and "gpu" variants of tensorflow-base
```

Comment:
Please reset tensorflow-base's build number to 0.

Reply:
Skip `py<37`, see https://github.com/tensorflow/tensorflow/blame/v2.10.0/tensorflow/tools/pip_package/setup.py#L314